README.md

```diff
@@ -57,3 +57,39 @@ If the file file you selected is supported by Nextcloud Text, Nextcloud Office o
 you will be able to collaborate on this file with all the call participants.
 
 You can then press "Close for all" if you want to hide the app for all participants.
+
+## Copy link to Clipboard and close the window/iframe
+
+This button will copy the desired link and can close the picker app window if it has been opened with a js `pickerWindow` button like 
+```
+newButton.onclick = () => {
+  pickerWindow = window.open('https://your.nextcloud.org/index.php/apps/picker/single-link?option=Clipboard', 'pickerWindow', 'popup');
+};
+```
+or can close the iframe if it has been opened with a js button similar to
+```
+newButton.onclick = () => {
+  // Create the iframe element
+  var pickerFrame = document.createElement('iframe';
```

Comment:
I think you should also be setting the iframe permission to allow it to use the clipboard, no? `allow="clipboard-write"`

Reply:
I don't know. Where I have tried this option, in Element for Nextcloud, it was working without it. If you think it will be needed and it is better to add it, I will. Let me know.